api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -173,4 +173,19 @@ public String getBaseCharacterSet() {
 	public void setBaseCharacterSet(String baseCharacterSet) {
 		this.baseCharacterSet = baseCharacterSet;
 	}
+	
+	/**
+	 * @return the nextSequenceValue
+	 */
+	public Long getNextSequenceValue() {
+	    if(nextSequenceValue == null) return new Long(- 1);
+	    return nextSequenceValue;
+	}
+
+	/**
+	 * @param nextSequenceValue : set the next identifier to be generated for this SequentialIdentifierGenerator
+	 */
+	public void setNextSequenceValue(Long nextSequenceValue) {
```

Comment:
In the revert, it was an Integer. Any reason for changing it to Long?

Reply:
There was some changes that came after the revert, nextSequenceValue is of type long across the entire codebase, should I convert everything to Integer instead?